sync/src/types/header_map/backend_sled.rs

```diff
@@ -27,8 +27,15 @@ impl KeyValueBackend for SledBackend {
         }
         .expect("failed to create a tempdir to save header map into disk");
 
-        let db: Db = sled::open(tmpdir.path())
+        // use a smaller system page cache here since we are using sled as a temporary storage,
+        // most of the time we will only read header from memory.
+        let db: Db = Config::new()
+            .mode(Mode::HighThroughput)
+            .cache_capacity(64 * 1024 * 1024)
```

Comment:
How about use `ByteUnit::Megabyte(64).as_u64()` here.

Reply:
I don't like to import a crate for this simple usage.